lib/OpenQA/CacheService/Task/Sync.pm

```diff
@@ -14,6 +15,8 @@ use constant RSYNC_RETRY_PERIOD => $ENV{OPENQA_RSYNC_RETRY_PERIOD} // 3;
 sub register ($self, $app, $conf) { $app->minion->add_task(cache_tests => \&_cache_tests) }
 
 sub _cache_tests ($job, $from = undef, $to = undef) {
+    my $ensure_task_retry_on_termination_signal_guard = OpenQA::Task::SignalGuard->new($job);
```

Comment:
```suggestion
    # this relies on constructor and destructor magic when the variable is getting out of scope
    my $ensure_task_retry_on_termination_signal_guard = OpenQA::Task::SignalGuard->new($job);
```
maybe add something like this as a comment?

Reply:
Such comment shouldn't be needed. The same is done in other places and the variable suffix "guard" should be sufficient to hint to that